dev-qt/qtwebengine/qtwebengine-5.15.9999.ebuild

```diff
@@ -84,13 +84,12 @@ RDEPEND="
 		~dev-qt/qtwidgets-${QTVER}
 	)
 "
-DEPEND="${RDEPEND}
+BDEPEND="${RDEPEND}
```

Comment:
that's not how `BDEPEND` works. Keep `DEPEND="${RDEPEND}`, and don't inherit either of them in `BDEPEND`.